### PR TITLE
[MODGOBI-212] - fix Module Descriptor and add descriptor validator

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
https://folio-org.atlassian.net/browse/MODGOBI-212
## Approach
 - Added module descriptor validator dependency
 - Fixed permissions

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -199,7 +199,17 @@
   ],
   "permissionSets": [
     {
-      "permissionName": "gobi.item.post",
+      "permissionName": "gobi.validate.item.get",
+      "displayName": "gobi - post order",
+      "description": "Creates an order"
```
Comment (Saba-Zedginidze-EPAM) [inline descriptors/ModuleDescriptor-template.json:204]: Display names and descriptions are same for new perm sets

**Review by Saba-Zedginidze-EPAM — APPROVED** [review]

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (SerhiiNosko) [inline descriptors/ModuleDescriptor-template.json:204]: displayName and description should be updated

Comment (azizbekxm) [reply descriptors/ModuleDescriptor-template.json]: Fixed

Comment (azizbekxm) [reply descriptors/ModuleDescriptor-template.json]: Fixed

**Review by SerhiiNosko — APPROVED** [review]

**Pull request merged**